src/main/kotlin/pl/droidsonroids/gradle/jacoco/testkit/JaCoCoTestKitPlugin.kt

```diff
@@ -11,6 +11,8 @@ class JaCoCoTestKitPlugin : Plugin<Project> {
 
     override fun apply(project: Project) {
         with(project) {
+            pluginManager.apply("jacoco")
+
```

Comment:
The plugins execution order in a `plugins` closure is not guaranteed and this plugin assumes `jacoco` was applied already.